Fabric.Terminology.SqlServer/Caching/CacheKeys.cs

```diff
@@ -6,9 +6,14 @@
 
     internal static class CacheKeys
     {
+        public static string ValueSetKey(string valueSetUniqueId)
+        {
+            return $"{typeof(ValueSet)}-{valueSetUniqueId}";
+        }
+
         public static string ValueSetKey(string valueSetUniqueId, IEnumerable<string> codeSystemCodes)
         {
-            return $"{typeof(ValueSet)}-{valueSetUniqueId}-{string.Join(string.Empty, codeSystemCodes)}";
+            return $"{ValueSetKey(valueSetUniqueId)}-{string.Join(string.Empty, codeSystemCodes)}";
```

Comment:
This is an interesting one - You're pointing out a possible error if the code system codes come in a different order.  I think a better approach would be to sort the code system codes first and then either join with commas (or not).  I don't see the benefit in serializing to JSON ... ?